apps/web/pages/_document.tsx

```diff
@@ -45,7 +45,8 @@ class MyDocument extends Document<Props> {
           <link rel="manifest" href="/site.webmanifest" />
           <link rel="mask-icon" href="/safari-pinned-tab.svg" color="#000000" />
           <meta name="msapplication-TileColor" content="#ff0000" />
-          <meta name="theme-color" content="var(--cal-bg)" />
+          <meta name="theme-color" media="(prefers-color-scheme: light)" content="cyan" />
```

Comment:
@sean-brydon for light mode is this the colour we want?

Reply:
definitely not cyan